src/components/aktivitetskrav/historikk/AktivitetskravHistorikk.tsx

```diff
@@ -51,9 +51,11 @@ export const AktivitetskravHistorikk = () => {
           <BodyShort size="small">{texts.subHeader}</BodyShort>
         </FlexColumn>
       </FlexRow>
-      {vurderinger.sort(byCreatedAt).map((vurdering, index) => (
-        <HistorikkElement key={index} vurdering={vurdering} />
-      ))}
+      <Accordion>
+        {vurderinger.sort(byCreatedAt).map((vurdering, index) => (
+          <HistorikkElement key={index} vurdering={vurdering} />
+        ))}
+      </Accordion>
```

Comment:
Betyr det at vi gjorde feil før, eller har det kommet en endring i `Accordion`-komponenten? Det har kanskje noe å si for focus/tabbing?

Reply:
Ja, vi gjorde nok feil før. Virker som det er lagt til litt ekstra border på Accordion som gjør at det ser rart ut slik vi brukte det før.